examples/apps/cra-demo/src/App.js

```diff
@@ -0,0 +1,70 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import TinyliciousClient from "@fluid-experimental/tinylicious-client";
+import { SharedMap } from "@fluidframework/map";
+
+const getContainerId = () => {
+    let isNew = false;
+    if (window.location.hash.length === 0) {
+        isNew = true;
+        window.location.hash = Date.now().toString();
+    }
+    const id = window.location.hash.substring(1);
+    return { id, isNew };
+};
+
+TinyliciousClient.init();
+
+const initFluidData = async (setData) => {
+    const containerConfig = {
+        name: 'cra-demo-container',
+        initialObjects: { myMap: SharedMap }
+    };
+
+    const { id, isNew } = getContainerId();
+    const fluidContainer = isNew
+        ? await TinyliciousClient.createContainer({ id }, containerConfig)
+        : await TinyliciousClient.getContainer({ id }, containerConfig);
+    setData(fluidContainer.initialObjects);
+}
+
+function App() {
+
+    const [fluid, setFluidData] = React.useState();
+    const [time, setViewTime] = React.useState('');
+
+    React.useEffect(() => {
+        if (fluid === undefined) {
+            // Get/Create container and set up our Fluid data structures
+            initFluidData(setFluidData);
+        } else {
+            // set up initial UI state
+            setTime(fluid.myMap.get("time"));
```

Comment:
```suggestion
            setViewTime(fluid.myMap.get("time"));
```
It looks like setTime gets declared later and doesn't accept any params